src/drive/mobile/modules/offline/duck.js

```diff
@@ -110,3 +112,8 @@ export const updateOfflineFileCopyIfNecessary = (
     await saveOfflineFileCopy(file, client)
   }
 }
+
+export const updateLocalStorageItem = async (key, value) => {
```

Comment:
It's more like an `addValueToExistingArrayInLocalStorage` than an `updateLocalStorageItem` right? 

Since this method is hard to name because she is very specific to the use case you are handling, I'm wonder if naming it `addFileIdToLocalStorageItem` is not more understandable